jobs/groundcrew/spec

```diff
@@ -157,3 +158,26 @@ properties:
     description: |
       Environment name you wish to group errors under in yeller.
     default: ""
+
+  tsa_host_key:
+     description: |
+       Must be specified, bosh can auto-generate (type ssh), see sample manifest.yml.
+       Only public key portion is needed here.
+     example:
+       private_key: |
+          -----BEGIN RSA PRIVATE KEY-----
+          ...
+          -----END RSA PRIVATE KEY-----
+       public_key: |
+          ssh-rsa ...
+
+  worker_key:
```

Comment:
This shouldn't be necessary; we already have `tsa.private_key`. I can just omit it when merging.